source/uwp/Renderer/lib/DefaultResourceDictionary.h

```diff
@@ -56,4 +56,62 @@ const PCWSTR c_defaultResourceDictionary = L"\
     <Style TargetType=\"Grid\" x:Key=\"ColumnSet\"> \
         <Setter Property=\"Margin\" Value=\"0,0,0,0\"/> \
     </Style> \
+\
+    <Style x:Key=\"HitTarget\" TargetType=\"Button\"> \
```

Comment:
>HitTarget [](start = 19, length = 9)

I wonder if we should rename this to SelectAction. In theory different hosts will want hit targets to be styled differently. We want to enable styling via resource dictionary, so we can document that to change the SelectAction behavior, they should use this key.

Alternatively for now we can ship with this, and require hosts to provide a custom renderer for all elements that support SelectAction if they want to change how it looks.